### PR TITLE
Add elevations in geovelo bike section

### PR DESCRIPTION
**Pull request by kadhikari**
For details: https://jira.kisio.org/browse/NAVP-1797

### CODE REVIEW

File: source/jormungandr/jormungandr/street_network/geovelo.py [hunk 1]
```diff
@@ -327,6 +327,13 @@ def _get_response(cls, json_response, pt_object_origin, pt_object_destination, f
             for sh in shape:
                 section.street_network.coordinates.add(lon=sh[0], lat=sh[1])
 
+            elevations = geovelo_section['details'].get('elevations')
+            if elevations:
```
Comment (azime) [inline source/jormungandr/jormungandr/street_network/geovelo.py]:
```suggestion
            elevations = geovelo_section.get('details', {}).get('elevations', [])
```

Comment (kadhikari) [reply source/jormungandr/jormungandr/street_network/geovelo.py]: :+1:

**Review by pbench — APPROVED** [review]

File: source/jormungandr/jormungandr/street_network/geovelo.py [hunk 2]
```diff
@@ -327,6 +327,13 @@ def _get_response(cls, json_response, pt_object_origin, pt_object_destination, f
             for sh in shape:
                 section.street_network.coordinates.add(lon=sh[0], lat=sh[1])
 
+            elevations = geovelo_section.get('details', {}).get('elevations', [])
+            if elevations:
+                for geovelo_elevation in itertools.islice(elevations, 1, sys.maxsize):
```
Comment (xlqian) [inline source/jormungandr/jormungandr/street_network/geovelo.py:332]: why not 
```suggestion
                for geovelo_elevation in elevations[1:]:
```

File: source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py [hunk 3]
```diff
@@ -199,6 +199,11 @@ def get_coordinate(self, obj):
             return None
 
 
+class ElevationSerializer(PbNestedSerializer):
+    distance_from_start = RoundedField(display_none=True)
+    elevation = RoundedField(display_none=True)
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py:204]: No coordinates?

Comment (kadhikari) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py:204]: No coordinates as decided

**Review by xlqian — APPROVED** [review]

**Review by azime — APPROVED** [review]

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**